### DISCUSSION TITLE
Error on saving the print page

### DISCUSSION THREAD

**Discussion by neha04** [pimcore/pimcore#8467] [Q&A / Support] [answered]
Hi 
I am getting following error on saving few print pages on the pimcore demo. Also for a few containers the pdf are not getting generated like for [this](http://demo.pimcore.fun/admin/login/deeplink?document_75_printcontainer) print container 

![Screenshot from 2021-03-16 09-04-30](https://user-images.githubusercontent.com/21308144/111252597-7d649880-8637-11eb-9e9e-537420cf41f8.png)

**Comment by fashxp** [comment, marked as answer]
Hi,
will be fixed with https://github.com/pimcore/web2print-tools/pull/16 as soon as we release Pimcore 6.9 (very soon) 

BR

**Reply by jeykodev** [reply]
@fashxp i know that the due date in the milestone was 16.03, but can we expect 6.9 in this week? Thanks ;)
